tests/cloud_test_logic/cloud_instance_mappings.py

```diff
@@ -43,16 +43,19 @@ def mock_get_indexes_response(indexes_list: Union[List[GetIndexesIndexResponseOb
     This decorator is used to mock the behavior of the requests.get which is used by
     MarqoCloudInstanceMappings object to retrieve and store specific information about
     cloud indexes: index_name, index status and index endpoint.
-    It allows you to set up mock responses for requests to the "/indexes" URL
-    while allowing real requests to other URLs.
+    It allows you to set up mock responses for requests to the "/indexes" URL.
+    Requests will be handled by side_effect function, which will return mock_get object
+    if url ends with "/indexes", and will call requests.get otherwise.
+    mock_get object can be argument of the decorated function, if to_return_mock is True.
```

Comment:
This is clearer. It can still be improved slightly:

```
if url ends with "/indexes" it returns a mock_get object, otherwise `requests.get` is called normally. 
A mock_get object is a MagicMock object with its return_value set to indexes_list.
```